src/lib/components/layout/Navigation.svelte

```
@@ -48,26 +53,150 @@
 		)}
 	>
 		<div class="flex h-full w-full items-center justify-between rounded-[15px]">
-			<a href={localizeHref(route('/'))} aria-label={m['nav.home']()}>
-				<enhanced:img
-					src="$assets/logo.avif"
-					alt={m['common.kesval_logo']()}
-					class="size-10 rounded-lg"
-				/>
-			</a>
+			<div class="flex items-center gap-4">
+				<a href={localizeHref(route('/'))} aria-label={m['nav.home']()}>
```

ESLint

[error] Found a link with a url that isn't resolved.

(svelte/no-navigation-without-resolve)